pkg/services/extsvcauth/models.go

```diff
@@ -4,16 +4,21 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 const (
 	ServiceAccounts AuthProvider = "ServiceAccounts"
-
-	// TmpOrgID is the orgID we use while global service accounts are not supported.
-	TmpOrgIDStr string = "1"
-	TmpOrgID    int64  = 1
 )
 
+func DefaultOrgID(cfg *setting.Cfg) int64 {
```

Comment:
nit: it's been reimplemented in multiple places, could you please move this to the root of `pkg/services/accesscontrol` (add to `pkg/services/accesscontrol/accesscontrol.go`)? And we'll use that in the future.

Reply:
I agree, I was thinking about this as well. I'll tackle it in another PR. Do you think having it in the `setting` package makes more sense?

Reply:
I think because the defaultOrgID mostly concerns the accesscontrol/authnz part I'd put it into the `accesscontrol` package.